client/layout/utils.ts

```diff
@@ -21,13 +21,14 @@ let pinnedSidebarTop = true;
 let pinnedSidebarBottom = false;
 let ticking = false; // Used for Scroll event throttling.
 
-export const handleScroll = (): void => {
+export const handleScroll = ( event: any ): void => {
+	console.log( event );
```

Comment:
We probably don't want to commit logging this event?

Reply:
Done eb3d162, thanks!